### ISSUE TITLE
Version on binder does not plot.

### ISSUE THREAD

**Issue by koldunovn** [intake/xrviz#70]
Currently, the version that is opened with the binder does not make the  plot, even if I click the `Plot` button.

**Comment by martindurant** [comment]
I believe this is the same as https://github.com/intake/xrviz/pull/69
I will try to make some time to put things right.

**Comment by martindurant** [comment]
Works OK now?

**Comment by rsignell-usgs** [comment]
I clicked the binder link, ran the notebook, got a plot, but then this:
https://www.youtube.com/watch?v=zWabYFZt-G8

**Comment by martindurant** [comment]
Constant unzooming! Is there an issue for this?

**Comment by rsignell-usgs** [comment]
Do you think it's a holoviews/geoviews issue or an xrviz issue?

**Comment by philippjfr** [comment]
Yeah something in holoviews. It's probably to do with aspect handling. Is this using recent HoloViews yet?

**Comment by martindurant** [comment]
The script uses "current" versions (whatever conda gives us from conda-forge on py37)

**Comment by martindurant** [comment]
@rsignell-usgs , if you still have this open, would be good to post the versions binder ended up using

**Comment by rsignell-usgs** [comment]
```
# packages in environment at /srv/conda/envs/notebook:
#
# Name                    Version                   Build  Channel
holoviews                 1.13.2             pyh9f0ad1d_0    conda-forge
geoviews                  1.8.1                      py_0    conda-forge
geoviews-core             1.8.1                      py_0    conda-forge
bokeh                     2.0.1            py37hc8dfbb8_0    conda-forge
hvplot                    0.5.2                      py_0    conda-forge
panel                     0.9.5                      py_1    conda-forge
```

**Comment by philippjfr** [comment]
Okay, so definitely latest, I'll try to reproduce.

**Comment by philippjfr** [comment]
Okay, I see the same thing. Will see if I can get a 1.13.3 release with a fix out very soon.

**Comment by koldunovn** [comment]
The plot works for me now on the binder, thanks!

**Comment by rsignell-usgs** [comment]
@philippjfr I'm so glad you could replicate the zooming out feedback loop issue and it sounds like you see the problem also.  Awesome!

**Comment by philippjfr** [comment]
Just wish we had an automated way to catch these kinds of regressions. Unfortunately testing client-server interactions in a browser is hard.
